tests/rules/bem-depth.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+var lint = require('./_lint');
+
+//////////////////////////////
+// SCSS syntax tests
+//////////////////////////////
+describe('bem depth - scss', function () {
+  var file = lint.file('bem-depth.scss');
+
+  // silly case to raise a warning for every selector/placeholder
+  it('[max-depth: 0]', function (done) {
+    lint.test(file, {
+      'bem-depth': [1, { 'max-depth': 0 }]
```

Comment:
I think that the value of an option shouldn't change the severity of the rule set by the user. E.g.
```
bem-depth:
  - 2
  - max-depth: 0
```
shouldn't silently turn off the rule. How about, instead:
- Specify max-depth > 0 in the docs
- Error if max-depth <= 0 inside this rule